src/sass/file-preview-card-group/stories/file-preview-card-group.stories.js

```diff
@@ -0,0 +1,343 @@
+export default { title: "Skin/File Preview Card Group" };
+
+const squareImage = () => `
+    <div class="file-preview-card">
+        <div class="file-preview-card__center">
+            <img src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-real-square-pic.jpg" alt="Image">
+        </div>
+        <div class="file-preview-card__menu-action">
+            <button class="icon-btn" type="button" aria-label="Open menu">
+                <svg class="icon icon--16" height="16" width="16" aria-hidden="true">
+                    <use href="#icon-overflow-vertical-16"></use>
+                </svg>
+            </button>
+        </div>
+    </div>
+`;
+
+const portraitImage = () => `
+    <div class="file-preview-card">
+        <div class="file-preview-card__center">
+            <img src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-profile-pic.jpg" alt="Image">
+        </div>
+        <div class="file-preview-card__menu-action">
+            <button class="icon-btn" type="button" aria-label="Open menu">
+                <svg class="icon icon--16" height="16" width="16" aria-hidden="true">
+                    <use href="#icon-overflow-vertical-16"></use>
+                </svg>
+            </button>
+        </div>
+    </div>
+`;
+
+const portraitWithCSSImageContainBackground = () => `
+    <div class="file-preview-card">
+        <div class="file-preview-card__center">
+            <span style="height: 100%; width: 100%; background-image: url('https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-profile-pic.jpg'); background-repeat: no-repeat; background-position: center; background-size: contain;"></span>
+        </div>
+        <div class="file-preview-card__menu-action">
+            <button class="icon-btn" type="button" aria-label="Open menu">
+                <svg class="icon icon--16" height="16" width="16" aria-hidden="true">
+                    <use href="#icon-overflow-vertical-16"></use>
+                </svg>
+            </button>
+        </div>
+    </div>
+`;
+
+const landscapeImage = () => `
+    <div class="file-preview-card">
+        <div class="file-preview-card__center">
+            <img src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-landscape-pic.jpg" alt="Image">
+        </div>
+        <div class="file-preview-card__menu-action">
+            <button class="icon-btn" type="button" aria-label="Open menu">
+                <svg class="icon icon--16" height="16" width="16" aria-hidden="true">
+                    <use href="#icon-overflow-vertical-16"></use>
+                </svg>
+            </button>
+        </div>
+    </div>
+`;
+
+const portraitDocWithTitle = () => `
```

Comment:
Please change all the PDF stories to match Skin as "other" than image - generic icon.